app/src/main/res/layout/activity_home.xml

```diff
@@ -7,15 +7,81 @@
     android:background="@color/white"
     tools:context=".HomeActivity">
 
+    <ImageView
+        android:id="@+id/imageView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="20dp"
```

Comment:
@raghav32 place values in dimen folder
and why marginTop of 178dp is used here?